### DISCUSSION TITLE
ADA compliance for PowerPoint

### DISCUSSION THREAD

**Discussion by jlindsteadt** [davidgohel/officer#499] [Ideas]
I may be misunderstanding the Alt-Text functionality, but it doesn't appear to be available for PowerPoint docs. Is that correct? If not, is it possible to add Alt-Text for both Flextables and ggplots? Below is a snippet of code attempting to add alt-text. Any help is appreciated. Thanks.

`library(officer)
library(flextable)
library(tidyverse)

x <- read_pptx()

data_flex <- flextable(iris)

y <- add_slide(x, layout='Title Slide') %>%
  ph_with(value=data_flex, 
          location=ph_location(left=0.5, top=0.5),
          alt_text = 'alt text test')

print(y, target='iris.pptx')`

**Comment by humoroussmile** [comment]
Could this be what you are hoping to do for ggplots?: [https://rdrr.io/cran/officer/man/external_img.html](https://rdrr.io/cran/officer/man/external_img.html)

This may not be helpful for `flextable`, I realize.

**Comment by jlindsteadt** [comment]
Thanks for the suggestion humoroussmile. The issue would be sometimes the ggplots need to be edited in the powerpoint, so pasting as an image wouldn't always be sufficient.
